Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -95,6 +96,33 @@ public NodejsProjectNode(NodejsProjectPackage package)
                     fileNode.Analyze();
                 }
             }
+
+            TryToAcquireTypings();
+        }
+
+        private bool ShouldAcquireTypingsAutomatically {
+            get {
+                return !_projectHasTypeScriptFiles
+                    && NodejsPackage.Instance.IntellisenseOptionsPage.EnableES6Preview
+                    && NodejsPackage.Instance.NpmOptionsPage.EnableAutomaticTypingsAcquisition;
+            }
+        }
+
+        private void TryToAcquireTypings() {
+            var controller = ModulesNode?.NpmController;
+            if (controller != null && ShouldAcquireTypingsAutomatically) {
+                var packages = controller.RootPackage.Modules.Where(package =>
+                        package.IsDependency
+                        || package.IsOptionalDependency
+                        || package.IsDevDependency
+                        || !package.IsListedInParentPackageJson);
+
+                TypingsAcquisition.AcquireTypings(
+                    controller.ListBaseDirectory,
+                    controller.RootPackage.Path,
+                    packages,
+                    null).ContinueWith(x => x);
```

Comment:
There is a warning if you don't consume the result of an async function. A noop continuation is one easy way to get around that.